config/engines.yml

```diff
@@ -30,7 +30,7 @@ bundler-audit:
     - Gemfile.lock
 csslint:
   image: codeclimate/codeclimate-csslint
-  description: Automated linting of Cascading Stylesheets
+  description: Automated linting of Cascading Stylesheets.
```

Comment:
This comes pretty much directly from the project page, and we try to copy/imitate official descriptions where possible.